### DISCUSSION TITLE
x-model for table row cell, is it possible?

### DISCUSSION THREAD

**Discussion by ericxin1982** [alpinejs/alpine#4113] [4. General]
Hi 

I have a json array for table, and x-for to html table, and right now I want to add input element for html table, each row has three columns, so has three input, and I want to use x-model for json array individual row, cell, is it possible?

Is it too slow?

Thanks
Eric Xin

**Comment by ekwoka** [comment]
It's possible and will work just as fast as if it wasn't in an x-for loop.

Totally fine and intended :)
